OneCommunityGlobal/HGNRest
# Guirong taking over for Luis - Unit test for getAllInvInProjectWBS inside InventoryController

# Description
Unit testing for Inventory Controller



## Related PRS (if any):
N/A

## Main changes explained:
First set of unit test for the Inventory Controller 
Created unit tests for getAllInvInProjectWBS function
Created a md file describing all positive and negative cases. 

## How to test:
1. check into current branch
2. do` npm install`
3. run `npm run test inventoryController`
4. verify all tests pass
## Note:

<img width="1228" alt="Screenshot 2024-05-12 at 7 51 44 PM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/22454953/d2d2de6e-cb47-4f6d-b606-dba4b891e08c">
